dev/docker/ocis-appprovider-onlyoffice/entrypoint-override.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+apk add curl
+
+#TODO: app driver itself should try again until OnlyOffice is up...
+
+retries=10
+while [[ $retries -gt 0 ]]; do
+    if curl --silent --show-error --fail http://onlyoffice/hosting/discovery > /dev/null; then
```

Comment:
I would suggest using `restart: always` in compose file like with `ocis-appprovider-collabora` service instead of new entrypoint. See https://github.com/saw-jan/oC-deployments/blob/master/ocis/wopi.yml

Reply:
it did not workout without enntry point using restart always.